lib/ui/auth/login_screen.dart

```diff
@@ -46,30 +50,43 @@ class LoginScreen extends StatelessWidget {
                     maxLines: 1,
                     controller: authController.passwordController,
                   ),
-                  TextButton(
-                    onPressed: () async {
-                      if (_formKey.currentState!.validate()) {
-                        isLoading.value = true;
-                        SystemChannels.textInput.invokeMethod('TextInput.hide');
-                        var result = await authController
-                            .signInWithEmailAndPassword(context);
-                        if (result) {
-                          isLoading.value = false;
-                          Get.to(HomeScreen());
-                        } else {
-                          isLoading.value = false;
-                          Get.snackbar(StringConstants.app_name,
-                              StringConstants.error_login_fail,
-                              snackPosition: SnackPosition.BOTTOM,
-                              duration: Duration(seconds: 3),
-                              backgroundColor:
+                  Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                    children: [
+                      TextButton(
+                        onPressed: () {
+                          Get.to(() => ForgotPasswordScreen());
```

Comment:
Could use Get.to(ForgotPasswordScreen());

Reply:
Get.to(ForgotPasswordScreen()); gives a warning in the console asking to change to `Get.to(() => ForgotPasswordScreen());`

Reply:
I get it :)